src/Microsoft.NetCore.Analyzers/Core/Runtime/SystemRuntimeAnalyzersResources.resx

```diff
@@ -384,9 +384,12 @@
   <data name="DisposeObjectsBeforeLosingScopeDescription" xml:space="preserve">
     <value>If a disposable object is not explicitly disposed before all references to it are out of scope, the object will be disposed at some indeterminate time when the garbage collector runs the finalizer of the object. Because an exceptional event might occur that will prevent the finalizer of the object from running, the object should be explicitly disposed instead.</value>
   </data>
-  <data name="DisposeObjectsBeforeLosingScopeMessage" xml:space="preserve">
+  <data name="DisposeObjectsBeforeLosingScopeNotDisposedMessage" xml:space="preserve">
     <value>In method '{0}', call System.IDisposable.Dispose on object created by '{1}' before all references to it are out of scope.</value>
   </data>
+  <data name="DisposeObjectsBeforeLosingScopeMayBeDisposedMessage" xml:space="preserve">
+    <value>In method '{0}', refactor the code to ensure that disposable object created by '{1}' is disposed on all paths where references to it go out of scope. If possible, wrap the creation within a 'using' statement or declaration. Otherwise, use a try-finally pattern, with a dedicated local variable declared before the try region and an unconditional Dispose invocation on non-null value in the 'finally' region, say 'x?.Dispose()'. If the object is explicitly disposed within the try region or the dispose ownership is transfered to another object or method, assign 'null' to the local variable just after such an operation to prevent double dispose in 'finally'.</value>
```

Comment:
Nit: "refactor" implies a transformation with no change in behaviour. Consider using "modify" or similar instead.

Reply:
I will make this change in the follow-up PR that I plan to send out later today to enable flagging exception paths in CA2000.